devDocs/developerGuide.t2t

```diff
@@ -845,6 +846,20 @@ NVDA will look for and execute an onUninstall function in installTasks.py when N
 After this function completes, the add-on's directory will automatically be removed.
 As this happens on NVDA startup before other components are initialized, this function cannot request input from the user.
 
+++ Optional plugin independent code ++
+In NVDA 2023.1 and above, it is possible to bundle plugin independent code with an add-on.
+This can be used to bundle base classes, constants, etc. that have to be available to multiple plugins in the add-on.
+This is especially useful if your add-on bundles an appModule and a globalPlugin and both plugins need to access the same class bundled with the add-on, for example.
```

Comment:
A detail: "especially" and "for example" do not seem so compatible in this phrasing. So only use one of them depending on what you want to say.

```suggestion
For example, this is useful if your add-on bundles an appModule and a globalPlugin and both plugins need to access the same class bundled with the add-on.
```

or:

```suggestion
This is especially useful if your add-on bundles an appModule and a globalPlugin and both plugins need to access the same class bundled with the add-on.
```